clash-prelude/src/Clash/Sized/Vector.hs

```diff
@@ -57,6 +57,7 @@ module Clash.Sized.Vector
   , merge
     -- * Modifying vectors
   , replace
+  , replace_int
     -- ** Permutations
```

Comment:
Ah, too bad this PR required the export of said function. Understandable though.

Reply:
As discussed in the other comment, I've removed this export.